### PR TITLE
Add information about user permissions to the CodeQL ADR

### PR DESCRIPTION
**Pull request by Izzzu**
Update the CodeQL ADR with information that alerts can be suppressed only by users with at least Write role. 
Users with Read role can only see the alerts in the PR.

### CODE REVIEW

File: docs/developer/decision-records/2022-02-11-codeql/README.md [hunk 1]
```diff
@@ -31,15 +31,23 @@ The results can be visible in the Github Workflow check view under the PR and in
 After clicking on the alert we can see a view with more detailed explanations about it, references and examples.
 
 ## Suppressing the alerts
-From both views (under the PS and in the Security Tab) the alerts can be analysed and dismissed/removed if they are not applicable.
+
+The alerts can be suppressed or removed by users with Security permissions which are assigned by default to user roles Write, Maintain and Admin.
+
+![CodeQL](security_permissions.png)
```
Comment (marcgs) [inline docs/developer/decision-records/2022-02-11-codeql/README.md:37]: nit: the circle highlighting the security setting is broken ;-)

**Review by marcgs — APPROVED** [review]

**Review by chrislomonico — COMMENTED** [review]
a few comments that may be non-issues

File: docs/developer/decision-records/2022-02-11-codeql/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (chrislomonico) [reply docs/developer/decision-records/2022-02-11-codeql/README.md:37]: works for me, it is when I preview the content in the PR (not the md but the preview itself; could do with pathing).  I will approve and if there is an issue we can address. When I view the branch, all is good (for the record)

**Review by chrislomonico — APPROVED** [review]
responded and approving

File: docs/developer/decision-records/2022-02-11-codeql/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Izzzu) [reply docs/developer/decision-records/2022-02-11-codeql/README.md:37]: double checked, sems to be ok

**Pull request merged**